bundles.config.js

```diff
@@ -325,8 +325,8 @@ exports.extensionBundles = [
   },
   {
     name: 'amp-experiment',
-    version: '0.1',
-    latestVersion: '0.1',
+    version: ['0.1', '1.0'],
+    latestVersion: '1.0',
```

Comment:
Latest version should be 0.1, 1.0 is still experimental.

Reply:
(I generally agree with this comment.) I noticed that `amp-story`, `amp-mustache`, and `amp-access-laterpay` all list the newest (experimental?) version as `latestVersion`. Should they too be rolled back until the newest version is ready for wide release?

https://github.com/ampproject/amphtml/blob/2b8b439203444545acd5c8780ae5ea79ac972b0a/bundles.config.js#L636-L638
https://github.com/ampproject/amphtml/blob/2b8b439203444545acd5c8780ae5ea79ac972b0a/bundles.config.js#L522-L524
https://github.com/ampproject/amphtml/blob/2b8b439203444545acd5c8780ae5ea79ac972b0a/bundles.config.js#L52-L54